src/R/Components/Impl/PackageManager/Implementation/View/DesignTime/DesignTimeRPackageViewModel.cs

```diff
@@ -1,24 +1,48 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System.Collections.Generic;
 using Microsoft.R.Components.PackageManager.Model;
 using Microsoft.R.Components.PackageManager.ViewModel;
 
 namespace Microsoft.R.Components.PackageManager.Implementation.View.DesignTime {
     internal class DesignTimeRPackageViewModel : IRPackageViewModel {
-        public DesignTimeRPackageViewModel(string name = null
+        public DesignTimeRPackageViewModel() {
+            Name = "ggplot2";
```

Comment:
It is design time code. Required for WPF designer to render something. Never executed in runtime. I think I can make it debug-only.